README.md

```diff
@@ -1,75 +1,99 @@
-[hash]: https://github.com/hashintel/hash/tree/main/packages/hash
-[blockprotocol.org]: https://blockprotocol.org/?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
 [þ hub]: https://blockprotocol.org/hub?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
+[block protocol]: https://blockprotocol.org/?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
+[block protocol documentation]: https://blockprotocol.org/docs?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
 [block protocol specification]: https://blockprotocol.org/docs/spec?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
+[blockprotocol.org]: https://blockprotocol.org/?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
+[becoming an embedder]: https://blockprotocol.org/docs/embedding-blocks?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
+[create a new block]: https://blockprotocol.org/docs/developing-blocks?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
 [core specification]: https://blockprotocol.org/docs/spec/core?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
+[developing blocks]: https://blockprotocol.org/docs/developing-blocks?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
+[discord]: https://blockprotocol.org/discord?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
 [graph service]: https://blockprotocol.org/docs/spec/graph-service?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
 [hook service]: https://blockprotocol.org/docs/spec/hook-service?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
-[block protocol documentation]: https://blockprotocol.org/docs?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
-[create a new block]: https://blockprotocol.org/docs/developing-blocks?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
-[developing blocks]: https://blockprotocol.org/docs/developing-blocks?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
-[using blocks]: https://blockprotocol.org/docs/using-blocks?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
-[becoming an embedder]: https://blockprotocol.org/docs/embedding-blocks?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
 [rfcs & roadmap]: https://blockprotocol.org/docs/spec/rfcs_and_roadmap?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
+[using blocks]: https://blockprotocol.org/docs/using-blocks?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_root
+[open issues]: https://github.com/blockprotocol/blockprotocol/issues?q=is%3Aissue+is%3Aopen
+[hash]: https://github.com/hashintel/hash/tree/main/packages/hash
+
+<!-- markdownlint-disable link-fragments -->
+
+[github_banner]: #block-protocol
```

Comment:
@nonparibus Markdownlint does not like link aliases for fragments:

```
$ markdownlint --dot --fix .
README.md:38 MD051/link-fragments Link fragments should be valid [Context: "#--what-is-the-block-protocol"]
README.md:42 MD051/link-fragments Link fragments should be valid [Context: "#--getting-started"]
README.md:50 MD051/link-fragments Link fragments should be valid [Context: "#--examples"]
README.md:56 MD051/link-fragments Link fragments should be valid [Context: "#--roadmap"]
README.md:63 MD051/link-fragments Link fragments should be valid [Context: "#--repository-overview"]
README.md:65:368 MD051/link-fragments Link fragments should be valid [Context: "[contributions](#contributing)"]
README.md:93 MD051/link-fragments Link fragments should be valid [Context: "#--contributing"]
README.md:111 MD051/link-fragments Link fragments should be valid [Context: "#--license"]
README.md:115 MD051/link-fragments Link fragments should be valid [Context: "#--license"]
README.md:119 MD051/link-fragments Link fragments should be valid [Context: "#--contact"]
README.md:127 MD051/link-fragments Link fragments should be valid [Context: "#--acknowledgments"]
error Command failed with exit code 1.
```

I disabled the rule for now.

Reply:
Eek, thanks for that. Is there an upstream issue we can track this disabling against?